sdk/communication/communication-media-composition/swagger/README.md

```diff
@@ -0,0 +1,41 @@
+# Azure Communication Services Media Composition Module
+
+> see [https://aka.ms/autorest](https://aka.ms/autorest)
+
+## Configuration
```

Comment:
I saw people often add autorest installation and command line here. Can be useful